docs/pages/versions/unversioned/sdk/image.mdx

```diff
@@ -74,3 +75,59 @@ import { Image } from 'expo-image';
 ```
 
 <APISection packageName="expo-image" apiName="Image" />
+
+## Generating a blurhash on a server
+
+Images can significantly improve the visual experience, but they can also slow down app/page loading times due to their large file sizes. To overcome this, we can create placeholder images using blurhash algorithm that can provide an immersive experience while deferring the loading of the actual picture until later.
+
+This brief guide will demonstrate how to create a blurhash of an uploaded image on the backend using `JavaScript` and `Express.js`. However, the same techniques and principles apply to other languages and server technologies.
+
+To simplify the process, we will install a few dependencies: [`multer`](https://github.com/expressjs/multer) for handling multipart requests, [`sharp`](https://github.com/lovell/sharp) for converting files to a data buffer, and the official [`blurhash JavaScript package`](https://github.com/woltapp/blurhash/tree/master/TypeScript).
+
+<Terminal cmd={['$ npm install multer sharp blurhash']} />
+
+Next, we need to import all needed functions from installed packages and initialized `multer`:
+
+```js
+// Multer is a middleware for handling `multipart/form-data`.
+const multer = require('multer');
+// Sharp allows you to recieve a data buffer from the uploaded image.
+const sharp = require('sharp');
+// Import the encode function from the blurhash package.
+const { encode } = require('blurhash');
+
+// Initilize `multer`.
+const upload = multer();
+```
+
+Assuming app is a variable that holds our express server, we can create an endpoint that accepts an image and returns a JSON response containing the generated blurhash.
```

Comment:
```suggestion
Assuming the `app` is a variable that holds our Express server, we can create an endpoint that accepts an image and returns a JSON response containing the generated blurhash.
```

Reply:
Adding to the above suggestion, I'd avoid using second person voice (We, Our, etc.)